### ISSUE TITLE
WAI-ARIA Combobox - Complete aria- attributes

### ISSUE THREAD

**Issue by sagalbot** [sagalbot/vue-select#866]
**Is your feature request related to a problem? Please describe.**
The `aria-explanded` & `combobox` attributes should be moved up to the `$refs.toggle` node. Currently they are set on the search input, but the toggle node houses information relevant to the combobox. 

Example from the spec:

```html
<div aria-label="Tag" role="combobox" aria-expanded="true" aria-owns="owned_listbox" aria-haspopup="listbox">
    <input type="text" aria-autocomplete="list" aria-controls="owned_listbox" aria-activedescendant="selected_option">
</div>
<ul role="listbox" id="owned_listbox">
    <li role="option">Zebra</li>
    <li role="option" id="selected_option">Zoom</li>
</ul>
```

**Describe the solution you'd like**
- [ ] pull up `role=combobox`
- [ ] pull up `aria-expanded`
- [ ] add all missing `aria-` tags

**Comment by igasparetto** [comment]
Any idea when this will this be done, please?

**Comment by adelriosantiago** [comment]
Are there any plans for this to be addressed?

**Comment by sagalbot** [comment]
@adelriosantiago I would for sure accept a PR.

**Comment by sagalbot** [comment]
Fixed in the v3.5.2 release.
